components/Comments.tsx

```diff
@@ -10,6 +11,42 @@ type Props = {
   currentLanguage?: Language;
 };
 
+const Comment = ({
```

Comment:
🙈 

I couldn't put links in images.ts since it's server side json?? so here's some horrible interpolation instead